packages/fx-core/tests/plugins/resource/apiconnector/expectedFiles/js/custom.js

```diff
@@ -0,0 +1,53 @@
+const teamsfxSdk = require("@microsoft/teamsfx");
+
+// The custom authProvider should implement the AuthProvider interface.
+// Here is a sample authProvider class. It will set custom property in the request header
+class CustomAuthProvider {
+  customProperty;
+  customValue;
+
+  constructor(customProperty, customValue) {
+    this.customProperty = customProperty;
+    this.customValue = customValue;
+  }
+
+  // Replace the sample code with your own logic
+  AddAuthenticationInfo = async (config) => {
+    if (!config.headers) {
+      config.headers = {};
+    }
+    config.headers[this.customProperty] = this.customValue;
+    return config;
+  };
+}
+
+// Loads current app's configuration and use app for auth.
```

Comment:
Let's remove `use app for auth` to avoid distracting users, since it's not AAD auth. Please also update the ts part